test/over_react/component_declaration/redux_component_test.dart

```diff
@@ -78,14 +78,14 @@ void main() {
       var renderedInstance = render(TestDefault()..store = store);
       TestDefaultComponent component = getDartComponent(renderedInstance);
 
-      store.actions.trigger1();
+      store.actions.trigger1(null);
```

Comment:
The argument to `ActionDispatcher` was originally optional but then was made required in built_redux 8.0.0